styleguide/source/assets/js/form-items.js

```diff
@@ -168,23 +169,19 @@
             {text: '', value: ''}
           ];
 
-          function selChange(){
-            var selection = $('#myCheckList').checkList('getSelection');
-
-            $('#selectedItems').text(JSON.stringify(selection));
-          }
-
-          if (typeof(jQuery.ui.checkList) != 'undefined'){
-            $('#filterList').checkList({
-              listItems: dataModel,
-              onChange: selChange
-            });
-          }
+          // I can't find where we're using this and it errors on Drupal end.
+          // function selChange(){
+          //   var selection = $('#myCheckList').checkList('getSelection');
+          //   $('#selectedItems').text(JSON.stringify(selection));
+          // }
+          //
+          // $('#filterList').checkList({
+          //   listItems: dataModel,
+          //   onChange: selChange
+          // });
```

Comment:
Let's ask the team if they know about this. I think I heard someone mention checkboxes the other day.

Reply:
Yeah, I get errors on the Drupal side!